prov/gni/include/gnix_ep.h

```diff
@@ -115,4 +193,33 @@ typedef ssize_t (*trecvmsg_func_t)(struct fid_ep *ep,
 				   const struct fi_msg_tagged *msg,
 				   uint64_t flags);
 
+/*
+ * inline functions
+ */
```

Comment:
Is there any reason you're not using `fr_free` and `_fr_alloc` defined in gnix_ep.c?